frontend/src/metabase/admin/settings/selectors.js

```diff
@@ -120,8 +123,12 @@ const SECTIONS = {
         ],
         getProps: setting => ({
           value: setting.value,
-          collectionFilter: collection =>
-            collection.personal_owner_id === null || collection.id === "root",
+          collectionFilter: (collection, index, allCollections) => {
+            return (
+              !isPersonalCollectionOrChild(collection, allCollections) ||
+              collection.id === "root"
```

Comment:
Does this simplification make sense?

```suggestion
              !isPersonalCollectionOrChild(collection, allCollections)
```

Reply:
I suppose so, since we basically just want to remove personal collections and their children